### PR TITLE
Add failing fixture test for RouteActionCallableRector on named routes

### PR DESCRIPTION
**Pull request by bradsi**
Related to Issue #59

### CODE REVIEW

File: tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc [hunk 1]
```diff
@@ -15,6 +15,7 @@ Route::match(['GET', 'POST'], '/users', 'SomeController@index');
 Route::fallback('SomeController@index');
 Route::options('/users', 'SomeController@index');
 Route::middleware([])->options('/users', 'SomeController@index');
+Route::get('/users', ['as' => 'users.index', 'uses' => 'SomeController@index']);
```
Comment (samsonasik) [inline tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc:18]: what if the structure flipped, `uses` key first, `as` key last, still applied?

Comment (bradsi) [reply tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc:18]: Still fails with the keys flipped

Comment (samsonasik) [reply tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc:18]: Could you add 1 more line for flipped usage to ensure all covered? Thanks

Comment (bradsi) [reply tests/Rector/StaticCall/RouteActionCallableRector/Fixture/fixture.php.inc:18]: Added new commit for flipped usage

**Comment by samsonasik** [comment]
@bradsi I cherry-picked your commit at PR https://github.com/rectorphp/rector-laravel/pull/61

**Pull request closed without merging**